src/Algebra/Graph.hs

```diff
@@ -473,8 +473,17 @@ foldg e v o c = go
 -- isSubgraphOf x y                         ==> x <= y
 -- @
 isSubgraphOf :: Ord a => Graph a -> Graph a -> Bool
-isSubgraphOf x y = overlay x y == y
-{-# SPECIALISE isSubgraphOf :: Graph Int -> Graph Int -> Bool #-}
+isSubgraphOf x y = AM.overlay (toAdjacencyMap x) yAM == yAM
```

Comment:
@nobrakal Well spotted!

Actually, we shouldn't even be using the `overlay`-based definition. The most efficient implementation is to simply call the `AM.isSubgraphOf` function after applying `toAdjacencyMap` to the arguments.

Reply:
Haha, you are right ! I just changed it !